web/packages/teleterm/src/ui/TopBar/Identity/IdentityList/IdentityList.tsx

```diff
@@ -19,68 +19,98 @@
 import { JSX } from 'react';
 import styled from 'styled-components';
 
-import { Box, Flex, Label, P3, Text } from 'design';
-import { ShieldCheck, ShieldWarning } from 'design/Icon';
+import { ButtonText, Flex, Label, P3 } from 'design';
+import { Logout, Refresh, ShieldCheck, ShieldWarning } from 'design/Icon';
 import Link from 'design/Link';
+import { Cluster } from 'gen-proto-ts/teleport/lib/teleterm/v1/cluster_pb';
 
-import { LoggedInUser } from 'teleterm/services/tshd/types';
-import { KeyboardArrowsNavigation } from 'teleterm/ui/components/KeyboardArrowsNavigation';
+import { ProfileStatusError } from 'teleterm/ui/components/ProfileStatusError';
+import { ProfileColor } from 'teleterm/ui/services/workspacesService';
 import { DeviceTrustStatus } from 'teleterm/ui/TopBar/Identity/Identity';
+import { RootClusterUri } from 'teleterm/ui/uri';
 
-import { IdentityRootCluster } from '../useIdentity';
-import { AddNewClusterItem } from './AddNewClusterItem';
-import { IdentityListItem } from './IdentityListItem';
+import { ColorPicker } from './ColorPicker';
+import {
+  AddClusterItem,
+  getClusterLetter,
+  IdentityListItem,
+  TitleAndSubtitle,
+} from './IdentityListItem';
 
-export function IdentityList(props: {
-  loggedInUser: LoggedInUser;
-  clusters: IdentityRootCluster[];
-  onSelectCluster(clusterUri: string): void;
-  onAddCluster(): void;
-  onLogout(clusterUri: string): void;
+export function ActiveCluster(props: {
+  activeCluster: Cluster | undefined;
+  activeColor: ProfileColor;
   deviceTrustStatus: DeviceTrustStatus;
+  onChangeColor(color: ProfileColor): void;
+  onRefresh(): void;
+  onLogout(): void;
 }) {
   return (
-    <Box minWidth="200px">
-      {props.loggedInUser && (
-        <>
-          <Flex px={3} pt={2} pb={2} justifyContent="space-between">
-            <Flex flexDirection="column" gap={2}>
-              <Text bold>{props.loggedInUser.name}</Text>
-              <Flex flexWrap="wrap" gap={1}>
-                {props.loggedInUser.roles.map(role => (
-                  <Label key={role} kind="secondary">
-                    {role}
-                  </Label>
-                ))}
-              </Flex>
-              <DeviceTrustMessage status={props.deviceTrustStatus} />
+    <>
+      <Flex p={3} pb={2} justifyContent="space-between">
+        <Flex flexWrap="nowrap" gap={2} flexDirection="column">
```

Comment:
One of these flexboxes seems unnecessary, no? The outermost `Flex` has only one child.

Reply:
Ion noticed it too, fixed already in https://github.com/gravitational/teleport/pull/51419/commits/68157572b75c9fae41858feb383e0b0dbeef3417